satellite/marketing/marketingweb/server.go

```diff
@@ -186,11 +190,23 @@ func formToStruct(w http.ResponseWriter, req *http.Request) (o marketing.NewOffe
 	return o,nil
 }
 
+func (s *Server) reRenderTables(w http.ResponseWriter){
+    req,err := http.NewRequest("GET","/",nil)
+    if err != nil {
+        s.log.Error("Rerender tables err", zap.Error(err))
+        s.serveInternalError(w,req,err)
+        return
+    }
+
+    http.Redirect(w,req,"/",http.StatusFound)
+}
+
+
 // Handler for POST requests to create, insert, and start a new offer or credit.
 func (s *Server) createOffer(w http.ResponseWriter, req *http.Request) {
 	o, err := formToStruct(w,req)
 	if err != nil{
-		s.log.Error("err from createFreeCredit Handler", zap.Error(err))
+		s.log.Error("create handler error", zap.Error(err))
```

Comment:
`parse form failed in createOffer`

Reply:
Right because if you're debugging it's more clear to get back exactly where it failed in the create  handler. I get it. Will do.